docusaurus.config.js

```diff
@@ -0,0 +1,143 @@
+// @ts-check
+
+const lightCodeTheme = require('prism-react-renderer/themes/vsLight');
+const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
+const creationYear = 2016;
+const dev = process.env.NODE_ENV === 'dev';
+const url = dev ? 'https://rapougnac.github.io/' : 'https://nyxx.l7ssha.xyz/';
```

Comment:
Probably better to have these sourced from an environment variable

Reply:
Got it!